packages/azure-kusto-data/package.json

```diff
@@ -29,26 +42,31 @@
         "directory": "azure-kusto-data"
     },
     "files": [
-        "source/**/*.js",
-        "source/**/*.d.ts",
-        "index.js",
-        "index.d.ts",
+        "dist/",
+        "dist-esm/src/",
+        "types/src/",
         "tsconfig.tsbuildinfo",
         "example.js",
-        "source/wellKnownKustoEndpoints.json"
+        "src/wellKnownKustoEndpoints.json"
```

Comment:
Is this still correct/do we still need this?
We do not have a src folder in our output anymore, and I think it is automatically copied to the dist-esm dir, if not - it should.

Reply:
Run "npm pack", it will be both at src/ and at dist-esm/src/, we don't need it in both places/

Reply:
didnt get your point - dist-esm is the build output its not there in the repo but the build copy it there .. same as will happen if you pack... it will copy it to the tar... no?